styles/components/HeroBanner.module.scss

```diff
@@ -0,0 +1,58 @@
+.HeroBanner {
+  @apply flex justify-between items-center bg-color-1 text-white p-8 h-[35em];
+}
+
+@media (max-width: 768px) {
```

Comment:
Hey Kara sorry for the multiple reviews here can you double check this is the correct way todo things with tailwind?

Reply:
I think I actually prefer the breakpoint approach here to tailwinds native approach. as long as amber and you approve I say we just use breakpoints